packages/rake-db/src/rakeDb.ts

```diff
@@ -115,6 +116,8 @@ const runCommand = async <C extends ColumnTypesBase = DefaultColumnTypes>(
     await generate(config, args.slice(1));
   } else if (arg === 'pull') {
     await pullDbStructure(toArray(options)[0], config);
+  } else if (arg === 'list' || arg === 'l') {
+    migrationList(config);
```

Comment:
await should be used, this can be called programmatically and the caller should be able to await this rakeDb function.